e2e/tests/functional/plugins/plot/overlayPlot.e2e.spec.js

```diff
@@ -198,3 +267,16 @@ async function getCanvasPixelsWithData(page) {
 
     return getTelemValuePromise;
 }
+
+/**
+ *
+ * @param {import('@playwright/test').Page} page
+ */
+async function assertLimitLinesExistAndAreVisible(page) {
+    await page.waitForSelector('.js-limit-area', { state: 'attached' });
+    const limitLineCount = await page.locator('.c-plot-limit-line').count();
```

Comment:
so you're declaring this count within the function and within the test with two separate assertions: 0 and 10.  You also use this locator on line 280.

I would consider maybe turning it into a 1 liner on all instances.

//Line limits should be 10 by default
expect(await page.locator('.c-plot-limit-line').count()).toBe(10)

Reply:
I agree, but I actually need the variable here for the loop so I can verify that the limit lines are actually visible on the page-- so I'll probably need to keep it in this case.